lib/shopify_api/resources/base.rb

```diff
@@ -28,6 +28,32 @@ def as_json(options = nil)
     end
 
     class << self
+      def search(params = {})
+        find(:all, params: params)
+      end
```

Comment:
this isn't a very efficient search - If I remember correctly won't this fetch records until something is found?